src/message/messageActionSheet.js

```diff
@@ -82,6 +86,21 @@ const reply = ({ message, dispatch, ownUser }) => {
 reply.title = 'Reply';
 reply.errorMessage = 'Failed to reply';
 
+const replyWithMention = ({ message, dispatch, ownUser, narrow, startReplyWithMention }) => {
+  const replyNarrow = getNarrowForReply(message, ownUser.user_id);
+  const replyMessage = `@**${message.sender_full_name}**`;
```

Comment:
One thing that this doesn't handle is the case where two users have the same display name — you might want to look at how the webapp code handles that.

Reply:
Aaah, I see, I need to use the `user_id`